website/docs/migration.md

```diff
@@ -0,0 +1,213 @@
+
+## Operation modes
+
+We offer multiple operation modes which help facilitate the testing of your application by enabling FerretDB to act as a proxy.
+For more details, refer to the [official documentation](./configuration/operation-modes.md).
+
+### Manual and automated testing with `diff-normal` mode
+
+For details on how to install FerretDB, refer to the [quickstart guide](../quickstart-guide/).
+
+You can manually test your application or use integration tests, among other methods.
+Afterward, you can inspect the differential output for errors or inconsistencies between responses that require your attention.
+
+As an example, let us say that your application performs some complex query and you'd like to test it in `diff-normal` mode.
+You would do the following:
+
+1. Start FerretDB in `diff-normal` mode.
+   This can be achieved by using the `--mode` [flag](./configuration/flags.md) or by setting the `FERRETDB_MODE` environment variable.
+2. Run `mongosh` and insert some documents.
+3. Run a query to fetch the first post from each author sorted by date and author.
+
+   Please note that due to running in `diff-normal` mode, any error returned from FerretDB will be transmitted to the client, allowing us to promptly identify the issue.
+   In the majority of cases, this does not necessitate additional scrutiny of the diff output.
+   Nevertheless, if FerretDB does not handle the error, additional inspection becomes necessary.
+
+   ```sh
+   # run mongosh
+   $ mongosh
+   ```
+
+   ```js
+   // insert some documents
+   db.posts.insertMany([
+     {
+       title: 'title A',
+       body: 'some content',
+       author: 'Bob',
+       date: ISODate('2023-08-29T10:33:23.134Z')
+     },
+     {
+       title: 'another title',
+       body: 'some content',
+       author: 'Bob',
+       date: ISODate('2023-08-28T10:33:23.134Z')
+     },
+     {
+       title: 'title B',
+       body: 'some content',
+       author: 'Alice',
+       date: ISODate('2023-08-20T10:33:23.134Z')
+     },
+     {
+       title: 'some other title',
+       body: 'some content',
+       author: 'Alice',
+       date: ISODate('2023-08-21T10:33:23.134Z')
+     }
+   ])
+
+   // run a query
+   db.posts.aggregate([
+     { $sort: { date: 1, author: 1 } },
+     {
+       $group: {
+         _id: '$author',
+         firstPost: { $first: '$date' }
+       }
+     }
+   ])
+   // the below error is returned to the client:
+   // MongoServerError: $group accumulator "$first" is not implemented yet
+   ```
+
+### Manual and automated testing with `diff-proxy` mode
+
+Continuing with the same example above, we can further examine the diff output while in `diff-proxy` mode.
+
+1. Run FerretDB in `diff-proxy` mode.
+   This can again be achieved by using the `--mode` [flag](./configuration/flags.md) or by setting the `FERRETDB_MODE` environment variable as already shown.
```

Comment:
```suggestion
   This can again be achieved by using the `--mode` [flag](configuration/flags.md) or by setting the `FERRETDB_MODE` environment variable as already shown.
```